app/views/thredded/shared/_topic_nav.html.erb

```diff
@@ -2,19 +2,21 @@
   <%= yield :thredded_breadcrumbs %>
 
   <ul class="thredded--topic-navigation">
-    <% if messageboard.try(:persisted?) %>
+    <% if messageboard.try(:persisted?) -%>
       <li class="thredded--topic-navigation--public">
-        <a href="<%= messageboard_topics_path(messageboard) %>">Topics</a>
+        <%= link_to 'Topics', messageboard_topics_path(messageboard) %>
       </li>
-    <% end %>
+    <% end -%>
+
     <li class="thredded--topic-navigation--private">
-      <a href="<%= private_topics_path %>">Private Topics
-        <% if (unread_count = unread_private_topics_count) > 0 %>
-            <span class="thredded--topic-navigation--unread">
-              <%= unread_private_topics_count %>
-            </span>
-        <% end %>
-      </a>
+      <%= link_to private_topics_path do %>
```

Comment:
If anything at this point, it's mostly about consistency and muscle memory. `grep -r "link_to" app/views` shows a whole lot of `link_to`'s, so ... I would err on the side of consistency. Not to be flippant, but - isn't half of rails an excessive abstraction? haha

Last but not least - vim/syntastic also yells at me when there are path helpers inside anchor tags :stuck_out_tongue: